recurser/string-is
# Add a UUID generator

![Generate_UUID](https://user-images.githubusercontent.com/96322/151188210-7fd29ed3-0467-4ca0-ae52-010d6cb43bd7.gif)



## How to test the PR

- Type `uuid` and check that a UUID is generated.
- Paste in a valid UUID and check that it is detected correctly, and the correct version is selected.
